qt/widgets/common/inc/MantidQtWidgets/Common/QtPropertyBrowser/CMakeLists.txt

```diff
@@ -20,19 +20,19 @@ set(QTPROPERTYBROWSER_SRCS
 # moc'd files will end up in build directory, so add to include path
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
-# Need to use a string of calls to the qt4_generate_moc macro instead of qt4_wrap_cpp because the latter produces files
+# Need to use a string of calls to the qt5_generate_moc macro instead of qt5_wrap_cpp because the latter produces files
 # with a cxx suffix when QtPropertyBrowser requires cpp
 
-qt4_generate_moc(src/qtpropertybrowser.h ${CMAKE_CURRENT_BINARY_DIR}/moc_qtpropertybrowser.cpp)
```

Comment:
Do we know how this worked until now? We removed using Qt4 2 years ago...

Reply:
The `qt4_generate_moc` command seems to generate the moc files as expected (same as the `qt5_generate_moc` command).

The conversation on [here](https://development.qt-project.narkive.com/odygyYSZ/installing-qt5config-cmake-from-the-qt-repo), and the code in this link seems to indicate that the `qt4_generate_moc` macro raps around the `qt5_generate_moc` macro

https://cmake.org/pipermail/cmake/2012-December/052970.html